css/style.css

```diff
@@ -23,12 +23,22 @@ body {
 .header,
 .portfolio-wrapper {
   width: 90%;
-  margin: 30px auto;
+  margin: 5% auto;
+  padding: 10px;
 }
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. :pray: :pray: 
- Please reduce the top margin of the navbar. :eyes: :eyes: 
Your design:
![image](https://user-images.githubusercontent.com/98363075/194716363-595a6e17-15b4-4048-9b58-0543d3bed2a4.png)
 Figma design:
![image](https://user-images.githubusercontent.com/98363075/194716382-47ecc31d-baf8-4b19-b8e2-1bd995df4268.png)

- Kindly reduce the space between the social media icons to stick to the Figma template. :eyes: :eyes: 
Your design:
![image](https://user-images.githubusercontent.com/98363075/194716502-2c47b359-c9aa-4faf-8742-d5f50fe1b9e3.png)
Figma design:
![image](https://user-images.githubusercontent.com/98363075/194716523-ceba37c2-4419-4cc7-91d2-146d4cb4e0b3.png)

- There is a huge black space at the end of your project. Kindly reduce it and please don't forget to reduce the space between the social media icons. :+1: :smiley: :smiley: 

Your design:
![image](https://user-images.githubusercontent.com/98363075/194716993-9731684d-be16-4ed8-87a9-2ca6cb8495aa.png)

Figma design:
![image](https://user-images.githubusercontent.com/98363075/194717036-09bf22cb-23df-4338-82ef-092d33626a27.png)